frontend/src/app/components/janus/janusVideoRoom.service.ts

```diff
@@ -234,13 +234,9 @@ export class JanusVideoRoomService {
         return this.stopAndStartSdiForwarding(shidurState, user, forwardIp, videoPorts[index], audioPort, changeTitle);
       });
 
-      this.$q.all(forwardPromises).then(() => {
-        this.$log.info('VideoRoom - all remote feeds forwarded successfully');
+      (<any> this.$q).allSettled(forwardPromises).then(() => {
```

Comment:
Yes. However, the system might be in a "stuck" situation. Stop worked but start didn't, so we've 3 out of 4 groups...